spec/requests/lettings_logs_controller_spec.rb

```diff
@@ -308,6 +306,13 @@
           expect(page).to have_link("Download (CSV, codes only)", href: "/lettings-logs/csv-download?codes_only=true")
         end
 
+        it "does not show a notification banner even if there are duplicate logs for this user" do
+          allow_any_instance_of(DuplicateLogsHelper).to receive(:duplicates_for_user).and_return({ lettings: [[1, 2]], sales: [] }) # rubocop:disable RSpec/AnyInstance
```

Comment:
We could use :duplicate trait, mock the scope instead and test a little bit more of the flow